### DISCUSSION TITLE
Update Ubuntu version of supabase/postgres Docker image

### DISCUSSION THREAD

**Discussion by pschaefer2305** [supabase/supabase#22024] [Feature Requests]
The current supabase/postgres is based on Ubuntu 20.04. It uses an old version of OpenSSL. For a self hosting of Supabase I want to use the pg_bm25 Postgres extension (https://www.paradedb.com). Using the current version I get the following error:

`FATAL:  could not load library "/usr/lib/postgresql/15/lib/pg_bm25.so": libssl.so.3: cannot open shared object file: No such file or directory`

It is related to the wrong OpenSSL version in Ubuntu 20.04. As Ubuntu 20.04 is getting older and older, I suggest to update the version to a newer LTS version. What do you think?

**Comment by olee** [comment]
Not only that.

Ubuntu 20 LTS is getting quite old by now so it would be very nice to get an updated image anyways.

**Reply by olee** [reply]
Any update on this? Being restricted to the outdated ubuntu version causes quite a lot of issues when trying to use other postgres extensions
